libs/application/templates/official-journal-of-iceland/src/components/advertPreview/AdvertPreview.tsx

```
@@ -0,0 +1,63 @@
+        </Box>
+      )}
+      {hasTypeOrSubject && (
+        <Box textAlign="center" marginBottom={advertText ? [2, 3, 4] : 0}>
+          {!!advertType && <Text variant="h3">{advertType}</Text>}
+          {!!advertSubject && <Text variant="h4">{advertSubject}</Text>}
+        </Box>
+      )}
+      {hasText && (
+        <Box
+          className={s.bodyText}
+          dangerouslySetInnerHTML={{ __html: advertText }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)